### ISSUE TITLE
Panic occurs when onedriver resumes after shutdown

### ISSUE THREAD

**Issue by mezuzza** [jstaf/onedriver#111]
Encountered an identical situation to #88. Seems that the situation wasn't totally fixed by #91.

```sh
$ onedriver --version
onedriver v0.9.2 f560cbe4
```

I resolved the issue personally using `sudo umount`, but I'm assuming that `fusermount -uz` would work just as well.
I've attached scrubbed logs which may help a little: [onedriver.log](https://github.com/jstaf/onedriver/files/6120625/onedriver.log)

I'm not 100% sure what caused the issue. There was at least one total power loss and a few suspend + resumes which is what I suspect caused the problem.

I think this issue should track just the panic itself, not the graceful handling of the panic. #92 already covers that and may functionally solve the issue with a retry after unmounting the existing file system.

**Comment by jstaf** [comment]
Looks like it crashed because one of the changes from the server was missing its checksum hashes from the server. Should be a relatively easy fix. :crossed_fingers: 

```
Mar 10 07:14:08 $HOSTNAME onedriver[2150]: panic: runtime error: invalid memory address or nil pointer dereference
Mar 10 07:14:08 $HOSTNAME onedriver[2150]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x6e3111]
Mar 10 07:14:08 $HOSTNAME onedriver[2150]: goroutine 29 [running]:
Mar 10 07:14:08 $HOSTNAME onedriver[2150]: github.com/jstaf/onedriver/fs.(*Cache).applyDelta(0xc000124240, 0xc000232280, 0x0, 0x0)
Mar 10 07:14:08 $HOSTNAME onedriver[2150]:         /build/onedriver-z7W8vq/onedriver-0.9.2/fs/delta.go:226 +0xaa1
Mar 10 07:14:08 $HOSTNAME onedriver[2150]: github.com/jstaf/onedriver/fs.(*Cache).DeltaLoop(0xc000124240, 0x6fc23ac00)
Mar 10 07:14:08 $HOSTNAME onedriver[2150]:         /build/onedriver-z7W8vq/onedriver-0.9.2/fs/delta.go:53 +0x2e9
Mar 10 07:14:08 $HOSTNAME onedriver[2150]: created by main.main
Mar 10 07:14:08 $HOSTNAME onedriver[2150]:         /build/onedriver-z7W8vq/onedriver-0.9.2/cmd/onedriver/main.go:119 +0x7bc
```

**Comment by jstaf** [comment]
Should be fixed in #113
